### ISSUE TITLE
Did we need to set HTTP_PLATFORM_PORT? using .scm CMD

### ISSUE THREAD

**Issue by sandakith** [AzureCAT-GSI/DropWizardOnAzure#1]
Try to figure out how the API App get the port, 
Did you manually set this to 80?
or do we need to set it at App Settings level?

I am getting java.lang.RuntimeException: java.net.SocketException: Address family not supported by protocol family: bind

**Comment by rsponholtz** [comment]
Dropwizard used port 80 by default.  If you want to use HTTPS for communication, you should still use port 80 and configure HTTPS in the web app.  I'll plan on putting in some instructions to that effect.  Also, the web app environment doesn't allow you binding to other ports (eg. the management port).  do get metrics, etc out of dropwizard, I'd suggest you leverage Azure Application insights.  I'm planning to add some instructions on that as well, but if you look at our [DevCamp](https://github.com/AzureCAT-GSI/DevCamp) repository, there is a java example for setting that up.

**Comment by sandakith** [comment]
Yeah, getting in to the configuring HTTPS now. Will try that out and let you know. Thank you.

**Comment by sandakith** [comment]
Update : We have the dropwizard API App service running with MS sql server now and API calls works fine too with the backend DB. We tried to get HTTPS working and here is what we did. We created a SSL certificate (self-signed to test), and uploaded it using the portal and update the web.config to force all traffic to HTTPS using a rule, however it seems out static content still does not work with basic auth. Also the certificate was not picked up rather it is now using a MS issued one which we never config to use. Any information relating to this would help. 

Also we tried AD but it blocked out all traffic ... If possible let us know if you have more information. Regarding basic auth and HTTPS with API Service. We also will look into this.
